spring-cloud-starter-stream-sink-router/src/test/resources/routertest-with-grab.groovy

```diff
@@ -0,0 +1,8 @@
+@Grab(group='org.codehaus.groovy', module='groovy', version='2.5.2')
```

Comment:
I guess this doesn't reflect reality because `org.codehaus.groovy:groovy` artifact dependency is here by default, pulled transitively by the `spring-integration-groovy`. 

Can't we use the same `GrooScript` for at least some string transformation?
Or something else for that repeat logic?

Reply:
Thanks, @Haybu !

But unfortunately your comment here is not related to the concern I've raised for this line of code.
Otherwise it might be considered as resolved and the original problem will remain.

Thanks for understanding.

Reply:
please let me know if you can think of a utility.

Reply:
I think I got one. Hope it LG now :)

Thanks